packages/server/graphql/types/DragEstimatingTaskPayload.ts

```diff
@@ -17,12 +19,22 @@ export const DragEstimatingTaskSuccess = new GraphQLObjectType<any, GQLContext>(
         return dataLoader.get('newMeetings').load(meetingId)
       }
     },
-    stageId: {
-      type: new GraphQLNonNull(GraphQLID)
+    stageIds: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLID)))
     },
-    stage: {
-      type: new GraphQLNonNull(EstimateStage),
-      resolve: resolveStage('ESTIMATE')
+    stages: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(EstimateStage))),
+      resolve: async ({meetingId, stageIds}, _args: unknown, {dataLoader}) => {
+        const meeting = await dataLoader.get('newMeetings').load(meetingId)
+        const {phases, teamId} = meeting
+        const phase = phases.find((phase: GenericMeetingPhase) => phase.phaseType === 'ESTIMATE')!
+        const {stages} = phase
+        const dbStages = stages.filter((stage: GenericMeetingStage) => stageIds.includes(stage.id))
+        dbStages.forEach((dbStage: GenericMeetingStage) =>
```

Comment:
-1 does this work? augmentDBStage is a pure function & you're running `forEach` instead of `map`